tests/Type/Symfony/Config/TreeBuilderTest.php

```diff
@@ -183,6 +168,21 @@ public function getProvider(): Iterator
                 ->end()
             ->end()
 		', 'Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition'];
+		yield ['
+		$arrayRootNode
+			->children()
+                ->arrayNode("methods")
+                    ->prototype("array")
+                        ->beforeNormalization()
+                            ->ifString()
+                            ->then(static function ($v) {
+								return [$v];
+							})
+                        ->end()
+                    ->end()
+                ->end()
+            ->end()
```

Comment:
While you're at it, could you please fix the indentation of this test case (plus the ones above that I forgot too)? 🙏 Would make it easier to understand.

Reply:
Done